.github/workflows/backup-kubernetes-databases-inhouse.yml

```diff
@@ -0,0 +1,168 @@
+        required: true
+        type: string
+      namespace:
+        description: 'Namespace where should be build app deployed'
+        required: true
+        type: string
+      url:
+        description: 'Kubernetes cluster url'
+        default: 'https://tkg.dev.bratislava.sk'
+        required: true
+        type: string
+    secrets:
+      service-account:
+        description: 'Kubernetes service account'
+        required: true
+      minio-endpoint:
+        description: 'Minio endpoint url'
+        required: true
+      minio-access:
+        description: 'Minio access key'
+        required: true
+      minio-secret:
+        description: 'Minio secret key'
+        required: true
+      minio-bucket:
+        description: 'Minio bucket name'
+        required: true
+      aws-key:
+        description: 'AWS key id'
+        required: true
+      aws-secret:
+        description: 'AWS secret access key'
+        required: true
+      aws-bucket:
+        description: 'AWS bucket name'
+        required: true
+
+jobs:
+  backup:
+    name: Backup databases
+    runs-on: ${{fromJSON(inputs.runs-on)}}
+    steps:
+      - name: Checking out
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Pipelines Version
+        run: |
+          echo "Pipelines version: 2.1.0"
+
+      - name: Directory check
+        run: pwd
+
+      - name: Kubectl tool installer
+        uses: Azure/setup-kubectl@v4.0.0
+
+      - name: Kubernetes set service account token
+        run: kubectl config set-credentials default --token=${{ secrets.service-account }}
+
+      - name: Kubernetes set server with certificate account token
+        run: kubectl config set-cluster ${{ inputs.cluster }}  --insecure-skip-tls-verify --server=${{ inputs.url }}
+
+      - name: Kubernetes set context cluster
+        run: kubectl config set-context ${{ inputs.cluster }} --cluster=${{ inputs.cluster }} --user=default
+
+      - name: Kubernetes use context
+        run: kubectl config use-context ${{ inputs.cluster }}
+      - name: Run database backup
+        run: |    
+          backup="${{ inputs.backup }}"
+          
+          if [[ $backup == "shared-wordpress" ]]; then
+            echo "We are going to backup shared wordpress databases."
+            select=$(kubectl get pod -n ${{ inputs.namespace }} --insecure-skip-tls-verify -l app.kubernetes.io/name=wordpress -o name)
+          else 
+            echo "We are going to backup postgres databases."
+            select=$(kubectl get pod -n ${{ inputs.namespace }} --insecure-skip-tls-verify -l service=database -o name)
+          fi
+          
+          mkdir "backup"
+          
+          for pod in ${select};
+          do
+            tmp=${pod##*/}
+            
+            if [[ $backup == "shared-wordpress" ]]; then
+              service=${tmp//-wordpress-*}
+            else
+              service=${tmp//-database-0}
```

Comment:
If you even want to get rid of these lines you could just use pod label `app=`, which should be defined for all of our deployments (I checked against PROD cluster). And if it isn't defined we want to add it.
Then you could just do 

```bash
select=$(kubectl get pod --namespace "${{ inputs.namespace }}" --insecure-skip-tls-verify --selector "$label" --output jsonpath='{.items[*].metadata.labels.app}')
```

which will give you, for `standalone` namespace, directly 
  - bratislava-strapi
  - city-account-strapi
  - clamav-cvdmirror
  - general-strapi
  - kupaliska-starz-be
  - nest-city-account
  - nest-clamav-scanner
  - nest-forms-backend
  - parking-pricing-api

Which seems to be exactly what you want here.